src/katgpucbf/xbgpu/engine.py

```diff
@@ -18,7 +18,7 @@
 This module defines the objects that implement an entire GPU XB-Engine pipeline.
 
 The XBEngine comprises multiple Pipeline objects that facilitate output data
-products. Additionally, the RxQueueItem and TxQueueItem objects, used in the
+products. Additionally, the InQueueItem and OutQueueItem objects, used in the
 XBEngine for passing information between different async processing loops,
 are defined here.
```

Comment:
There is no OutQueueItem class, and it's the *classes* that are defined here. Additionally, "async processing loops" sounds like event loops, which is confusing.

```suggestion
products. Additionally, several subclasses of :class:`.QueueItem` are defined for passing
data between parts of the engine.
```